mds/factories.py

```diff
@@ -637,6 +637,7 @@ class Meta:
         )
     )
     saved_at = datetime.datetime(2018, 8, 1, 1, tzinfo=pytz.utc)
+    first_saved_at = datetime.datetime(2018, 8, 1, 1, tzinfo=pytz.utc)
```

Comment:
`factory.SelfAttribute('saved_at')` to keep consistency when changing the value in tests.